src/views/Portal/components/ApplicationForm/index.view.tsx

```diff
@@ -82,7 +94,66 @@ const ApplicationForm: React.FC = () => {
 
   const saveData = () => {}
 
-  const submitData = () => {}
+  const submitData = async () => {
+    try {
+      const bodyData = new FormData()
+      bodyData.append("fname", contactFormData.fname)
+      bodyData.append("lname", contactFormData.lname)
+      bodyData.append("email", contactFormData.email || "none")
+      bodyData.append("phone", contactFormData.phone)
+      bodyData.append("age", demographicFormData.age)
+      bodyData.append("pronounCount", "1")
+      for (let i = 0; i < demographicFormData.pronouns.length; i += 1) {
+        bodyData.append("pronouns[0-N-1]", demographicFormData.pronouns[i])
+      }
+      bodyData.append("sexualityCount", "1")
+      for (let i = 0; i < demographicFormData.sexuality.length; i += 1) {
+        bodyData.append("sexuality[0-N-1]", demographicFormData.sexuality[i])
+      }
```

Comment:
Just as a note: pronounCount should be number items in the array and the data appended is each item in the array. Same for sexuality.

Reply:
It should look something more like this
`bodyData.append("pronounCount", integertostr(demographicFormData.pronouns.length)`
`bodyData.append('pronoun[${i}]', demographicFormData.pronouns[i]`)